libnd4j/include/ops/declarable/headers/transforms.h

```diff
@@ -41,7 +41,7 @@ namespace nd4j {
         DECLARE_CONFIGURABLE_OP(invert_permutation, 1, 1, false, 0, 0);
         #endif
 
-        DECLARE_CUSTOM_OP(concat, -1, 1, false, 0, -2);
+        DECLARE_CUSTOM_OP(concat, -1, 1, false, 0, 1);
```

Comment:
Why enforce it? We can just assume default axis as 0

Reply:
this is due to  compatibility with tf
there is no default value for axis input argument in corresponding tf operation